ui/admin/app/templates/scopes/scope/edit.hbs

```diff
@@ -3,13 +3,30 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
+{{#if @model.isGlobal}}
+  {{page-title (t 'resources.global.title')}}
+  <Breadcrumbs::Item
+    @text={{t 'resources.global.title'}}
+    @icon='globe'
+    @route='scopes.scope.edit'
+    @model={{@model.id}}
+  />
+{{/if}}
+
 <Rose::Layout::Page as |page|>
 
   <page.breadcrumbs>
     <Breadcrumbs::Container />
   </page.breadcrumbs>
 
   <page.header>
+    {{#if @model.isGlobal}}
+      <h2>
+        {{t 'resources.global.title'}}
+        {{! TODO: Add doc link }}
```

Comment:
None blocking question: are we waiting for the link to be provided? when adding the doclink I do recommend to not use the rose component `<DocLink>` but a regular HDS link, [here an example](https://github.com/hashicorp/boundary-ui/pull/2037/files#diff-d0edf5ba3ad67a92b84cd9ef362c648a9b07cb0e25a434c5b9f425a90d726fe2R13-R15)